modules/python/python-startup.c

```diff
@@ -369,11 +369,11 @@ _py_activate_venv(void)
 
   gchar *python_venv_binary = g_build_path(G_DIR_SEPARATOR_S, python_venv_path, "bin", "python", NULL);
   wchar_t *python_programname = Py_DecodeLocale(python_venv_binary, NULL);
-  g_free(python_venv_binary);
 
   msg_info("python: activating virtualenv",
```

Comment:
On a side note, the same message on line 223 (when using python >=3.8) is using `msg_debug`.  This look inconsistent but I did not changed it here.  My feeling is that both messages should be using `msg_debug`.

Reply:
You are right that both should be msg_debug() would be great to change that too.

In any case I'm approving this, good catch. Thanks

Reply:
Done, thanks!